network-api/networkapi/wagtailpages/pagemodels/buyersguide/products.py

```diff
@@ -876,17 +875,13 @@ def product_type(self):
         return "unknown"
 
     def get_context(self, request, *args, **kwargs):
-        BuyersGuidePage = apps.get_model(app_label='wagtailpages', model_name='BuyersGuidePage')
```

Comment:
Removing the logic that was used in order to pass the home page into the context